### ISSUE TITLE
Request: A script that pulls the transifex ".po" files and combine them into one

### ISSUE THREAD

**Issue by thetaung2** [OpenDevelopmentMekong/wp-odm_theme#946]
By downloading .po file from one transifex project (where translations has already been done) and uploading into a new project, strings that are already translated will be automatically updated in the new project. It'd improves both efficiency and consistency of localization work.

**Comment by Huyeng** [comment]
Combination them all in one file is really good. 
However, it is really concerned that the translated string won't display correctly for some English words that has different meaning in another language for different context.

**Comment by thetaung2** [comment]
Can you elaborate @Huyeng ? I guess we can download ".po" files for each language so there's no need to worry about another language? Or you mean one English word might have different meanings in different context even in a single language?

**Comment by Huyeng** [comment]
Well, I refereed a single English word has different meaning in local language for different context. 
A simple example is,  the word Landscape has different meaning in a view or a layout.

Actually, I just raised my concern on that, if we can ensure that there is no case like this, I'm fine with this, 
because My editor team used to request to combine all translating strings in one file for translation when they noticed that some words or strings are repeatedly translated. 

I have a question would like to clarify what are the po files that would like to combine together, all CKAN po files or will including with WP?
